isaaclab_arena/embodiments/agibot/agibot.py

```diff
@@ -0,0 +1,262 @@
+                        0.0,
+                        -0.7071,
+                        0.0,
+                    ],  # rpy: [0, -90, 0] to make the gripper direction from +Z to +X
+                ),
+            ),
+        ],
+    )
+
+    def __post_init__(self):
+        # Add a marker to the end-effector frame
+        marker_cfg = FRAME_MARKER_CFG.copy()
+        marker_cfg.markers["frame"].scale = (0.1, 0.1, 0.1)
+        marker_cfg.prim_path = "/Visuals/FrameTransformer"
+        self.ee_frame.visualizer_cfg = marker_cfg
+
+
+@configclass
+class AgibotRightArmSceneCfg(AgibotLeftArmSceneCfg):
+    """Scene configuration for the Agibot right arm."""
+
+    def __post_init__(self):
+        # update the ee_frame for the right arm
+        self.ee_frame = FrameTransformerCfg(
+            prim_path="{ENV_REGEX_NS}/Robot/base_link",
+            debug_vis=False,
+            target_frames=[
+                FrameTransformerCfg.FrameCfg(
+                    prim_path="{ENV_REGEX_NS}/Robot/right_gripper_center",
+                    name="right_end_effector",
+                    offset=OffsetCfg(
+                        pos=[0.0, 0.0, 0.0],
+                    ),
+                ),
+            ],
+        )
+
+
+@configclass
+class AgibotLeftArmActionsCfg:
+    """Action configuration for the Agibot robot."""
+
+    arm_action = RMPFlowActionCfg(
+        asset_name="robot",
+        joint_names=["left_arm_joint.*"],
+        body_name="gripper_center",
+        controller=AGIBOT_LEFT_ARM_RMPFLOW_CFG,
+        scale=1.0,
+        body_offset=RMPFlowActionCfg.OffsetCfg(pos=[0.0, 0.0, 0.0], rot=[0.7071, 0.0, -0.7071, 0.0]),
+        articulation_prim_expr="/World/envs/env_.*/Robot",
+        use_relative_mode=True,
+    )
+
+    gripper_action = mdp.AbsBinaryJointPositionActionCfg(
+        asset_name="robot",
+        threshold=0.5,
+        joint_names=["left_hand_joint1", "left_.*_Support_Joint"],
+        open_command_expr={"left_hand_joint1": 0.994, "left_.*_Support_Joint": 0.994},
+        close_command_expr={"left_hand_joint1": 0.0, "left_.*_Support_Joint": 0.0},
+    )
+
+
+@configclass
+class AgibotRightArmActionsCfg:
+    """Action configuration for the Agibot robot."""
+
+    arm_action = RMPFlowActionCfg(
+        asset_name="robot",
+        joint_names=["right_arm_joint.*"],
+        body_name="right_gripper_center",
+        controller=AGIBOT_RIGHT_ARM_RMPFLOW_CFG,
+        scale=1.0,
+        body_offset=RMPFlowActionCfg.OffsetCfg(pos=[0.0, 0.0, 0.0]),
+        articulation_prim_expr="/World/envs/env_.*/Robot",
+        use_relative_mode=True,
+    )
+
+    gripper_action = mdp.AbsBinaryJointPositionActionCfg(
+        asset_name="robot",
+        threshold=0.5,
+        joint_names=["right_hand_joint1", "right_.*_Support_Joint"],
+        open_command_expr={"right_hand_joint1": 0.994, "right_.*_Support_Joint": 0.994},
+        close_command_expr={"right_hand_joint1": 0.0, "right_.*_Support_Joint": 0.0},
+    )
```

Comment:
As above. Suggestion to make a single class which takes an `ArmMode` as input and constructs the config struct appropriately.